source/sass/views/youtube-regrets-2022.scss

```diff
@@ -0,0 +1,50 @@
+.youtube-regrets-2022 {
+  .recommendations {
+    background-color: $black;
+    padding-top: 85px;
+
+    &__title {
+      font-size: 30px;
+      line-height: calc(34 / 30);
```

Comment:
**Question** Is there a reason these are so specific? Couldn't we just round them to the next round size from the scale in the [design system](https://www.figma.com/file/JT9Dvdyc96AqQ3HJlnA9Qq/WIP-Mozilla-Foundation-Design-Style-Library?node-id=1495%3A793) (100%, 110% 120%, 130%, 140%, 150%)?

Reply:
Ok. No worries then. I was not aware of the heavy reuse approach.